docs/user/alerting/action-variables.asciidoc

```diff
@@ -5,405 +5,230 @@
-          }
-        }
-      }
-    ]
-  }
-}
-----
-
-You can create the following Mustache template in the email action for your rule:
-
-[source,js]
-----
-# {{context.title}} <1>
-
-{{#FormatDate}} {{{date}}} ; America/New_York {{/FormatDate}} <2>
-
-{{context.conditions}}
-
-**documents** _[view in Discover]({{{context.link}}})_ <3>
-
-| provider | action | duration | <4>
-| -------- | ------ | -------- |
-{{#context.hits}}{{#_source.event}}| {{provider}} | {{action}} | {{#duration}}{{#EvalMath}} round( {{{duration}}} / 1000 / 1000 / 1000 ) {{/EvalMath}} sec{{/duration}} {{^duration}}-n/a-{{/duration}} |{{/_source.event}}
-{{/context.hits}}
-----
-<1> Renders the value of the `context.title` variable as a level 1 heading.
-<2> Renders the value of the `date` variable as a formatted date in the America/New_York time zone.
-<3> Shows examples of `**bold**`, `_italic_`, and `[text](url)` links.
-<4> Shows a table with three columns, with one row per element in the `context.hits` array.
-From each of those elements, you can access the `provider`, `action`, and `duration` fields of the `_source.event` object.
-The `duration` field is rendered as a number of seconds, rounded to the nearest second.
-It's stored as nanoseconds so it needs to be divided by a billion to yield seconds.
-The duration field is optional, so you can use a `{{#duration}} ... {{/duration}}` section to render the duration if it's present and show `-n/a-` otherwise.
-
-When rendered into Markdown and then HTML and viewed in an email client, it looks like this:
-
-[role="screenshot"]
-image:images/email-mustache-template-rendered.png[Email template rendered in an email client]
+[[alert-actiongroup-variable]]`alert.actionGroup`::
+(string) The ID of the action group of the alert that scheduled the action. For example, `query matched`.
+[[alert-actiongroupname-variable]]`alert.actionGroupName`::
+(string) The name of the action group of the alert that scheduled the action. For example, `Query matched`.
+[[alert-actionsubgroup]]`alert.actionSubgroup`::
+(string) The action subgroup of the alert that scheduled the action.
+//TBD When does the subgroup appear? Is it related to the subcategory of ServiceNow connectors?
+`alert.consecutiveMatches`::
+(integer) The number of consecutive runs that meet the rule conditions.
+`alert.flapping`::
+(boolean) A flag on the alert that indicates whether the alert status is changing repeatedly.
+[[alert-id-variable]]`alert.id`::
+(string) The ID of the alert that scheduled the action. For example, `query matched`.
+`alert.uuid`::
+(string) A universally unique identifier for the alert. While the alert is active, the UUID value remains unchanged each time the rule runs. For example, `27ace796-1dfb-4db9-9ce1-7a10472fd461`. preview:[]
+`context`::
+(object) The context of the alert, which varies by <<rule-types,rule type>> and typically contains multiple variables.
+////
+//TBD: Is there a reason these don't appear in the UI variable list?
```

Comment:
Anything in `state` is essentially persisted variables for the rule.  We probably never should have exposed these in the first place, but they could have some diagnostic value.  I think a statement somewhere that `state` exists and contains stateful data regarding the rule is all we need here ...